keyworker-specs/src/test/groovy/uk/gov/justice/digital/hmpps/keyworker/model/TestFixture.groovy

```diff
@@ -31,30 +31,25 @@ class TestFixture {
 
     def loginAs(UserAccount user, int kwFrequency) {
         currentUser = user
-        oauthApi.stubValidOAuthTokenRequest()
-        browser.to LoginPage
```

Comment:
we now don't show a login page, just bypass it altogether and return the callback